docs/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+
+#### Account
+
+Accounts allow you to keep separate records of your DCR funds. Transferring DCR across accounts will create a transaction on the blockchain.
+
+#### Agenda voting
+
+See [consensus rules voting](#consensus-rules-voting).
+
+#### Address
+
+The cryptographic hash of a given public key. Addresses are publicly viewable on the blockchain and can be used to send and receive transactions, similar to an email address. 
```

Comment:
Addresses can not send, only receive.

Reply:
That is better I think and much closer, but it's not really a hash of the public key either.  Perhaps: "The address is a human-readable representation of the public key receiving payment."

Reply:
Using that:

#### Address

A secure identifier that represents a possible destination for a payment, similar to an email address. The address is a human-readable representation of the public key receiving payment.